provisioning/roles/qa_framework/schema.j2

```diff
@@ -0,0 +1,26 @@
+{% macro expand_ansible_connection_attributes(attributes_map) %}
+{% for attr_key, attr_value in attributes_map.items() %}
+{% if 'ansible' in attr_key %}
+{{- attr_key }}: {{ attr_value }}
+{% endif %}
+{% endfor %}
+{%- endmacro -%}
+
+{% if qa_framework is defined%}
+qa_framework:
+  hosts:
+  {%- for qa_framework_key, qa_framework_value in qa_framework.items() %}
+
+    qa_framework{{ loop.index }}:
+      ansible_host: {{ qa_framework_value['ip'] }}
+      {{ expand_ansible_connection_attributes(qa_framework_value) | indent(6) }}
+      qa_repository_reference: {{ qa_framework_value['qa_repository_reference'] }}
+  {%- endfor %}
+{% endif %}
+
+
```

Comment:
Use always the same number of line breaks (either 1 or 2). Apply this to all jinja files.

Reply:
Done d0a4e404e7dceb25c4e78663b78bfca9e8a7c00a